boards/shields/adafruit_data_logger/doc/index.rst

```diff
@@ -0,0 +1,77 @@
+.. _adafruit_data_logger_shield:
```

Comment:
Can you put the .jpg through a size minifier? 98KB is large

Reply:
Done.